Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM php:5.6-cli
```

Comment:
Should we be using our own base here? something a bit lighter like graze/composer.